articles/quickstart/spa/angular-next/index.yml

```diff
@@ -0,0 +1,50 @@
+title: Angular
+image: /media/platforms/angular.png
+author:
+  name: Steve Hobbs 
+  email: steve.hobbs@auth0.com
+  community: false
+alias:
+  - angular
+language:
+  - Javascript
+framework:
+  - Angular
+topics:
+  - quickstart
+contentType: tutorial
+useCase: quickstart
+show_releases: true
+sdk:
+  name: auth0-angular
+  url: https://github.com/auth0/auth0-angular
+snippets:
+  dependencies: application-platforms/angular2/dependencies
+  setup: application-platforms/angular2/setup
+  use: application-platforms/angular2/use
+seo_alias: angular
+default_article: 01-login
+articles:
+  - 01-login
+show_steps: true
+github:
+  org: auth0-samples
+  repo: auth0-angular-samples
+  branch: next
+requirements:
+  - Angular 9+
```

Comment:
I thought we were targetting 10+ with the SDK. Shouldn't we do that here as well?

Reply:
The SDK should work with at least the last 2 majors, so 9 and 10 in this case.

We **build** with 10, but the minimum requirement that we support is 9 (which is LTS until August next year). So I put 9 here as that's how I read what this requirement field is for.

Reply:
Makes sense. Thanks for clarifying.